.github/workflows/cd.yml

```diff
@@ -9,6 +9,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Get branch name
+        id: vars
+        run: echo ::set-output name=branch::${{ github.event.pull_request.head.ref }}
```

Comment:
This is for every branch you create. You need to change it to work only for master

Reply:
okay